### ISSUE TITLE
Strange Graphical issues; Error loading font texture

### ISSUE THREAD

**Issue by turkeybacontimmy** [LoneGazebo/Community-Patch-DLL#2162]
Bug Report Template

Mod Version : Most recent, July 7th release, version 1.7.7 ; errors also occurred in previous version

1. Mod List (if using standard CPP set, leave blank): IN ADDITION to full Vox Populi, withOUT EUI:
(JEM) Nuclear Missile Range Extension
Fortress Borders
Global - City State Airbases
Global - City State Overseas Territory
Global - No Conquered Allies
Global Warming
Global Warming Congress
Improved Air Recon
Improvement - Airbases
InfoAddict
JFD's : Cities In Development; Cultural Diversity (Core); Exploration Continued Expanded; Rise to Power
More Frequent World Congress
Quick Turns
R.E.D. : Modpack; Xtreme; and Smaller Landmarks
Reforestation
Units - Missiles No Supply
Units - Super Carrier
World Congress Reformation

I know it's long. I used to play with even more, and no issues would arise; none of these mods have ever caused an issue, which is why I am confused and believe it to be an issue in the new versions of Community Balance Project.

2. Type of error (i.e. crash, interface bug, AI quirk):  Upon founding a city, I encounter crazy interface glitches. It has been different in each case (I have been working for days trying to figure the issue out). Generally, there has been issues of the City Banner being completely black, along with every other art icon in the interface and Civilopedia. In addition, in some cases the City View will have a horribly glitched hex grid and interface. A few days ago, these were the only issues, however now upon founding the first city I also get four popups of an error which reads "Error loading font texture." 

3. Steps to reproduce:  Simply set up a new game, and found your first city.

4. Additional information:  Strangely enough, in my attempts to figure out the issue, there have been times when no error occurred at all, and others in which the entire interface is broken. The other day I planned on taking screenshots to post an issue here, and nothing went wrong. However, going to play today it broke again.

I must also note that for months, almost a year, I have been playing with this same collection of mods with absolutely NO issue; it is only since I bought a new computer that these issues have occurred. I went back the other day and updated all of the mods to the latest versions of everything, and still the problem persists.

I have also tried verifying the game cache through Steam, as well as deleting all files in the cache folder as some sources explain to do.

---------------------------
Supporting information:  I am trying to add the Database.log, lua.log, and minidump files; however, each time I am told that these file types are not supported. I have trying compressing everything into a single zip file as well, but when I try adding it I am told the same thing.

Please note that you can attach .zip files to Issues by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files: Database.log and Lua.log needed
[Lua.log.docx](https://github.com/LoneGazebo/Community-Patch-DLL/files/360350/Lua.log.docx)

[Database - Copy.log.docx](https://github.com/LoneGazebo/Community-Patch-DLL/files/360349/Database.-.Copy.log.docx)

2. Minidump file (located in your Civ5.exe directory) Could not add, unsupported file type (even in a .zip folder) I apologize

3. Screenshots (if needed)
![20160712143853_1](https://cloud.githubusercontent.com/assets/20389013/16785221/bd2bb342-4841-11e6-9059-db66687c48fc.jpg)
![20160712143855_1](https://cloud.githubusercontent.com/assets/20389013/16785224/c0a1dc68-4841-11e6-9a1f-caa5b54f8362.jpg)
![20160712143903_1](https://cloud.githubusercontent.com/assets/20389013/16785228/c3a954ea-4841-11e6-901a-e12089e7e13c.jpg)
![20160712143926_1](https://cloud.githubusercontent.com/assets/20389013/16785229/c53eb3a4-4841-11e6-83b5-32b9a6c149e7.jpg)
![20160712143941_1](https://cloud.githubusercontent.com/assets/20389013/16785230/c682e30c-4841-11e6-85f7-a614f7547054.jpg)

**Comment by Vyyt02** [comment]
First of all, you should try without the additional mods! :)

The screenshots are too small, I cannot see much in them, but it seems similar to the issue some people are having with Nvidia cards. See my reply in this thread:
https://github.com/LoneGazebo/Community-Patch-DLL/issues/2163

And yes, these can also start to appear after several months of playing without any glitches.

**Comment by LoneGazebo** [comment]
Please post your Lua log. If logging is not enabled, enable it, and try a game/reproduce the issue, and then post logs.

**Comment by turkeybacontimmy** [comment]
Vyyt , I actually did try with just the CBP installed and the same issues occurred haha.

Gazebo, sorry; I posted the Lua.log file I found, I must not have logging enabled. 

In any case, I figured out the problem, so thank you guys for the assistance; thankfully it was ultimately unnecessary, but it is so very much appreciated nonetheless! I had to disable a couple graphics functions and lower one setting to medium, and it runs fantastic and perfect again!

And Gazebo, thank you so much for your work. Your mods have made Civilization 5 my favorite game again, since I found it. I appreciate it so much, thank you!
